test/integration/component/test_ldap_autoImport.py

```diff
@@ -0,0 +1,588 @@
+from marvin.cloudstackTestCase import cloudstackTestCase
```

Comment:
@sarathkouk Few comments:
- Missing license, fix that.
- Please also cleanup code and use pep8 to fix the styling issues.
- Fix the filename (i.e. lower case and naming pattern) to adhere to the convention used in other test files.
- Any new dependency such as `ddt` etc that are not already in marvin's dependency list, must be added to marvin's dependency list.
- Also suggest how we can include this test in `blueorangutan`, what are the additional requirements/dependencies